### DISCUSSION TITLE
Default padding guidance on card organisms?

### DISCUSSION THREAD

**Discussion by kylegilbreath** [twilio-labs/paste#435] [General] [answered]
Heyo friends 👋! I had a discussion topic around whether or not we want to provide some guidance on what the card container padding should be so we are maintaining consistency on similar pages (i.e. setting type of pages that have a cards for each setting group). For the sake of discussion, let's call this combination of components (card + heading + paragraph + button) an organism. Should we be using a consistent default padding for similar organisms from page to page throughout the same product to maintain consistency?

I know we were purposeful in keeping the card flexible to allow for the end user to easily change the default padding, which I do support. But for these similar organisms, should we be more explicit about which default padding to use, and only advise changing that padding if necessary for a specific purpose.

The complete card sketch symbol has a default padding of 20px, which I've been using throughout my designs, but noticing that some PDs are playing around with this padding in some of their designs and wanted to raise the concern before we get too far along as we start pulling these new components into our designs. I've attached a few pages for reference of potentially similar type of card organisms. 

Anyway, would love to get some other people's thoughts here! Thanks!

![Settings_configured](https://user-images.githubusercontent.com/12417517/81414018-98a37500-9103-11ea-8a69-7b90e1c58bb0.png)
![Populated](https://user-images.githubusercontent.com/12417517/81414026-9b05cf00-9103-11ea-9cac-52d52940d906.png)
![BP_BusinessProfiles_Index_empty_state Copy](https://user-images.githubusercontent.com/12417517/81414028-9b9e6580-9103-11ea-85f0-c5cc7aa4a6ff.png)

**Comment by mindytom** [comment]
A consistent default padding sounds like a great idea.

To clarify @kylegilbreath, you're suggesting to have 20px padding between any of the combinations of headings, paragraphs, cards, and buttons?

**Reply by kylegilbreath** [reply]
I'm suggesting we stick with a default padding container for the card (the current default is 20px but I'm fine with any padding value here, more just want a consistent value), regardless of what might be contained within it. Unless there is a specific reason, or card type (i.e. maybe a centered promo-type of card for empty pages). I'm advocating for adding guidance around this so maintain a consistent feel throughout the product.

**Comment by JT1989** [comment]
we've  (PN eng + design) been starting to experiment with the 2-column grid with the gutter as a guide. Example: 

<img width="476" alt="Screen Shot 2020-05-08 at 10 06 35 AM" src="https://user-images.githubusercontent.com/8630856/81742251-a7b45b00-9454-11ea-9b6d-b2967e219221.png">

Used this grid: https://share.goabstract.com/2d429ee0-fc03-41aa-9a9b-260d25c21023?sha=77e48f6ae28910495d0dc3209d854923ac6ea637

**Comment by serifluous** [comment, marked as answer]
Thanks for this discussion, everyone! Having more consistent default spacing across the components is something we know could be better. We'll be working on it soon 🤞

For cards specifically, are you all seeing patterns when people are using non-default padding? Does it tend to show up for a particular kind of use case?

Can you tag more folks in this thread who are using the non-default padding? Even if you can get them to just drop an Abstract link here when you come across it in crit, that'd be really helpful! We want to get as many people to chime in here, so we can nail down guidelines without being overly restrictive.
